### PR TITLE
Allow for indexless installs

### PR DESCRIPTION
**Pull request by scollazo**
This adds the ARCHIVEMATICA_<service>_SEARCH_ENABLED env var

### CODE REVIEW

**Comment by scollazo** [comment]
This is needed by artefactual/deploy-pub#52

**Review by sevein — CHANGES_REQUESTED** [review]
Can we add a note to the README? LGTM.

**Review by jrwdunham — APPROVED** [review]
Yes, with mention in the README this looks good to me.

**Comment by scollazo** [comment]
Updated README to tell about the new `archivematica_src_search_enabled` var, and to point to the README.md files in archivematica.git

File: README.md [hunk 1]
```diff
@@ -56,6 +56,10 @@ The default Archivematica 1.7 logging sends the events to the standard streams,
 
 The log file sizes and the directories to store the logs are configurable for each service. The default values can be found in [`defaults/main.yml`](defaults/main.yml).
 
+Disable ElasticSearch use
```
Comment (sevein) [inline README.md]: `s/ElasticSearch/Elasticsearch`

File: README.md [hunk 2]
```diff
@@ -56,6 +56,10 @@ The default Archivematica 1.7 logging sends the events to the standard streams,
...
+-------------------------
+
+The default Archivematica install relies on ElasticSearch for different features (Archival storage, Backlog and Appraisal tabs). If you need to disable them, the boolean environment variable `archivematica_src_search_enabled` has to be set to `False`
```
Comment (sevein) [inline README.md]: Nitpick: `archivematica_src_search_enabled` isn't an environment variable. Role variable?

**Comment by jrwdunham** [comment]
@sevein What do you think? Can this be merged now?

File: README.md [hunk 3]
```diff
@@ -56,6 +56,10 @@ The default Archivematica 1.7 logging sends the events to the standard streams,
 
 The log file sizes and the directories to store the logs are configurable for each service. The default values can be found in [`defaults/main.yml`](defaults/main.yml).
 
+Disable Elasticsearch use
+-------------------------
+
+The default Archivematica install relies on ElasticSearch for different features (Archival storage, Backlog and Appraisal tabs). If you need to disable them, the role variable `archivematica_src_search_enabled` has to be set to `False`
```
Comment (sevein) [inline README.md]: s/ElasticSearch/Elasticsearch

**Review by sevein — APPROVED** [review]
Small cosmetic suggestion but I think it could be fixed later if we're in a rush. 👍

**Comment by jrwdunham** [comment]
I fixed the typo. I also used this role in a deploy-pub Vagrant deploy and confirmed it can be used to correctly deploy AM headlessly. See https://github.com/artefactual-labs/archivematica-acceptance-tests/pull/67#issuecomment-365447477.

**Pull request merged**